collections/_development_standards/platform-integration.md

```diff
@@ -272,3 +272,43 @@ For local development, it is useful to log to File.
 ### References
 
 * [App Service local cache size limits](https://docs.microsoft.com/en-us/azure/app-service/overview-local-cache#how-the-local-cache-changes-the-behavior-of-app-service)
+
+## ASP.NET Core Health Checks
+
+ASP.NET Core provides built in features to easily configure health check endpoints.  All apps should implement a basic health check with a /ping endpoint that establishes whether an app has started.  In addition, more sophisticated health checks can be implemented with a /health endpoint.
+
+The /ping endpoint will be used by the Application Gateway to establish whether the application is available, if it does not return a status code in the 200-399 range then the application will be marked as unhealthy and the Application Gateway will not route traffic to it.  By [default](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-3.1#customize-the-http-status-code-1) Health Checks will return a 200 if the application is healthy.  An alert will be posted to the #das-alerts Slack channel in the event that a staus code outside the acceptable range is returned.
+
+### Example
+Adding health checks to the Startup class, they can be added to Startup extensions in a similar manner.  
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddHealthChecks()
+        .AddCheck<SomeHealthCheck>("Checks the health of something");
```

Comment:
You could add in a couple of concrete examples

Reply:
Added some more examples